beam/io-pubsub/src/main/java/cz/o2/proxima/beam/io/pubsub/PubSubDataAccessor.java

```diff
@@ -18,8 +18,7 @@
 import com.google.common.base.Preconditions;
 import cz.o2.proxima.beam.core.DataAccessor;
 import cz.o2.proxima.beam.core.io.StreamElementCoder;
-import cz.o2.proxima.direct.pubsub.proto.PubSub;
-import cz.o2.proxima.pubsub.shaded.com.google.protobuf.InvalidProtocolBufferException;
+import cz.o2.proxima.direct.pubsub.util.PubSubUtils;
```

Comment:
I don't think we should make beam operator dependent direct on direct operator. We must either pull this code to module that will be used by both direct and beam (located at root/io, as mentioned) or just copy&paste that.

Reply:
completely refactored